packages/components/src/toggle-group-control/toggle-group-control-option/component.tsx

```diff
@@ -54,23 +77,26 @@ function ToggleGroupControlOption(
 		className,
 		isActive && styles.buttonActive
 	);
+	const optionLabel = radioProps[ 'aria-label' ] ?? label;
```

Comment:
The null coalescing operator here will allow an empty string in the `aria-label` prop to be used for the `optionLabel`. Not a big deal, I noticed this while debugging the Storybook examples.

In the case of an empty string would it be better to fall back to the `label` value?
```suggestion
	const optionLabel = !! radioProps[ 'aria-label' ]
		? radioProps[ 'aria-label' ]
		: label;
```

Reply:
No strong opinions here, but I had the same question when introduced and @ciampo suggested to have this check: https://github.com/WordPress/gutenberg/pull/35423#discussion_r724760813.